starburst-presto/README.MD

```diff
@@ -18,6 +18,6 @@ You can find more information about using initialization actions with Dataproc i
 ## Important notes
 * This script must be updated based on which Presto version you wish to install
 * You may need to adjust the memory settings in `jvm.config` based on your needs
-* Presto is set to use HTTP port `8080` by default
+* Presto is set to use HTTP port `8070` by default
```

Comment:
Default in code is 8060:

```suggestion
* Presto is set to use HTTP port `8060` by default
```

Reply:
@mattsfuller are we OK changing the default?

@ChethanUK if 8080 is a wrong default, we should change this for both presto and starburst-presto the same way. This may be one commit for both.

Reply:
I don't think there is a wrong default. It's conflicting with another optional component that is also using 8080. The same argument could be made that the other thing is using the wrong default.

I think the best solution is to make the port number configurable and passed in to override the default.

Reply:
> both presto and starburst-presto the same way.

Okay, Will update in presto as well :)